src/plugins/orangehrmRecruitmentPlugin/config/routes.yaml

```
@@ -16,11 +16,18 @@ apiv2_recruitment_vacancies:
 apiv2_recruitment_vacancy_attachment_vacancyId:
   path: /api/v2/recruitment/vacancy/attachments/{vacancyId}
   controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
```

```suggestion
apiv2_recruitment_vacancies_attachments:
  path: /api/v2/recruitment/vacancies/{vacancyId}/attachments
  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
```
:thinking:  is this getAll

---

yeah one vacancy can have number of vacancyAttachments

---

```yaml
apiv2_pim_employees_attachment:
  path: /api/v2/pim/employees/{empNumber}/screen/{screen}/attachments/{id}
  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
  methods: [ GET, PUT ]
  defaults:
    _api: OrangeHRM\Pim\Api\EmployeeAttachmentAPI
  requirements:
    empNumber: '\d+'

apiv2_pim_employees_attachments:
  path: /api/v2/pim/employees/{empNumber}/screen/{screen}/attachments
  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
  methods: [ GET, POST, DELETE ]
  defaults:
    _api: OrangeHRM\Pim\Api\EmployeeAttachmentAPI
  requirements:
    empNumber: '\d+'
```
somewhat similar patter there in PIM attachments

---

done